python/shopping.py

```diff
@@ -0,0 +1,57 @@
+def main():
+    sum_of_buy = 0
+    shopping = {}
+    x = 0
+    price_so_far = 0
+    while(x == 0):
+
+        number = int(input(
+            "Enter 1 for buing stuff, 2 for removing them or 3 if you are done with the shopping:"))
+        if(number == 3):
+            x = 1
+            print("thank you for buing the total cost is:", price_so_far)
+
+        if(number == 1):
+
+            prodact_name = input(
+                "what is the name of the prodact thet you want?\n")
+            prodact_price = int(input("what is the price?\n"))
```

Comment:
Same comment as before - don't cast to int before verifying that it actually is an integer.